README.md

```diff
@@ -150,26 +210,71 @@ When using the webpack dev server, which does not go through Rails, bootstrap is
+### Linters
+The React on Rails generator can add linters and their recommended accompanying configurations to your project. There are two classes of linters: ruby linters and JavaScript linters.
 
-This is how you actually render the React components you exposed to `window` inside of `clientGlobals` (and `global` inside of `serverGlobals` if you are server rendering).
+##### JavaScript Linters
+JavaScript linters are **enabled by default**, but can be disabled by passing the `--skip-js-linters` flag (alias `j`) , and those that run in Node have been add to `client/package.json` under `devDependencies`.
 
-`react_component(component_name, props = {}, options = {})`
-+ **react_component_name:** Can be a React component, created using a ES6 class, or `React.createClass`, or a generator function that returns a React component.
-+ **props:** Ruby Hash which contains the properties to pass to the react object
-+ **options:**
-  + **generator_function:** default is false, set to true if you want to use a generator function rather than a React Component.
-  + **prerender:** enable server-side rendering of component. Set to false when debugging!
-  + **trace:** set to true to print additional debugging information in the browser. Defaults to true for development, off otherwise.
-  + **replay_console:** Default is true. False will disable echoing server-rendering logs to the browser. While this can make troubleshooting server rendering difficult, so long as you have the default configuration of logging_on_server set to true, you'll still see the errors on the server.
-+ Any other options are passed to the content tag, including the id
+##### Ruby Linters
+Ruby linters are **disabled by default**, but can be enabled by passing the `--ruby-linters` flag when generating. These linters have been added to your Gemfile in addition to the the appropriate Rake tasks.
 
-`def server_render_js(js_expression, options = {})`
+#### Running the Linters
+To run the linters (runs all linters you have installed, even if you installed both Ruby and Node):
 
-This is a helper method that takes any JavaScript expression and returns the output from evaluating it. If you have more than one line that needs to be executed, wrap it in an IIFE. JS exceptions will be caught and console messages handled properly.
+```bash
+rake lint
+```
+
+Run this command to see all the linters available
+
+```bash
+rake -T lint
+```
+
+**Here's the list:**
+```bash
+rake lint               # Runs all linters
+rake lint:eslint        # eslint
+rake lint:js            # JS Linting
+rake lint:jscs          # jscs
+rake lint:rubocop[fix]  # Run Rubocop lint in shell
+rake lint:ruby          # Run ruby-lint as shell
+rake lint:scss          # See docs for task 'scss_lint'
+```
 
-## Developing with Webpack Dev Server
+## Node Dependencies and NPM
+### Updating
+After installing the files, you may want to update the node dependencies. This is analogous to updating gem versions:
```

Comment:
this does not belong in the react_on_rails readme as it's about node and npm.

great stuff for the generated client/README.md as well as the docs directory!